src/Classes/Factories/SortApplicantsControllerFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Don't need this because the sort applicants controller is not necessary, you can delete it